whisper/src/builder.rs

```diff
@@ -85,7 +85,7 @@ impl WhisperBuilder {
         Ok(metadata)
     }
 
-    pub fn build(self, path: impl AsRef<Path>) -> Result<WhisperFile, BuilderError> {
+    pub fn build<P: AsRef<Path>>(self, path: P) -> Result<WhisperFile, BuilderError> {
```

Comment:
what's wrong with `impl Trait`?

Reply:
just to be consistent. IMHO, this looks cleaner.